### PR TITLE
Add secDecimalDigits to readme and CLI help

### PR DESCRIPTION
**Pull request by icyflame**
- Also, fix the `compact` part of the CLI help

### CODE REVIEW

**Comment by kevva** [comment]
I think this option needs a better name if it were to be included.

File: cli.js [hunk 1]
```diff
@@ -7,19 +7,23 @@ var prettyMs = require('./');
...
 		'',
 		'Examples',
 		'  $ pretty-ms 1337',
 		'  1.3s',
 		'  $ pretty-ms 1337 --verbose',
 		'  1.3 seconds',
-		'  $ ~1s'
+		'  $ pretty-ms 1337 --compact',
+		'  ~1s',
+		'  $ pretty-ms 1337 --secDecimalDigits 4',
```
Comment (sindresorhus) [inline cli.js]: `--secDecimalDigits` => `--sec-decimal-digits`

`meow` will convert it to camelCase.

File: cli.js [hunk 2]
```diff
@@ -7,19 +7,23 @@ var prettyMs = require('./');
...
-		'  $ pretty-ms <milliseconds> [--compact] [--verbose]',
+		'  $ pretty-ms <milliseconds> [--compact] [--verbose] [--secDecimalDigits <number>]',
 		'  echo <milliseconds> | pretty-ms',
 		'',
 		'Options',
-		'  --compact  Only show the first part',
-		'  --verbose  Use full-length units',
+		'  --compact                      Only show the first part',
+		'  --verbose                      Use full-length units',
+		'  --secDecimalDigits <number>    Number of digits to appear after the seconds decimal point',
```
Comment (sindresorhus) [inline cli.js]: I thik `<number>` is moot here at it's shown in the above notation.

File: readme.md [hunk 3]
```diff
@@ -79,20 +79,26 @@ $ npm install --global pretty-ms
 ```
 $ pretty-ms --help
 
+  Convert milliseconds to a human readable string: 1337000000 → 15d 11h 23m 20s
```
Comment (sindresorhus) [inline readme.md]: I usually drop this as it's moot.

File: readme.md [hunk 4]
```diff
@@ -48,7 +48,7 @@ Milliseconds to humanize.
 
 #### options
 
-##### secDecimalDigits
+##### sec-decimal-digits
```
Comment (sindresorhus) [inline readme.md]: Nope, as I said it's converted to camelCase, so this one should be camelCase.

Comment (icyflame) [inline readme.md]: Sorry, I used `sed` and it replaced all the occurences! Fixed now.

Comment (sindresorhus) [inline readme.md]: Nope, as I said it's converted to camelCase, so this one should be camelCase.

Comment (sindresorhus) [inline readme.md]: Kinda unrelated, but do you have any suggestion on a shorter option name for this than `secDecimalDigits`?

**Comment by sindresorhus** [comment]
Thanks :)

File: readme.md (hunk 4 above)
Comment (icyflame) [inline readme.md]: I guess we can use `dec-digits` ? `secDecimalDigits` is great for the API,
but on the command line, typing something that long, well, it takes longer,
is all!

Comment (sindresorhus) [inline readme.md]: Could maybe add some short-flags then.

Comment (icyflame) [inline readme.md]: Sure. Only for `secDecimalDigits` ? (`verbose` and `compact` are fine in their present form, right?)

Comment (sindresorhus) [inline readme.md]: They could all have one.

Comment (icyflame) [inline readme.md]: Do you mean short tags as in `-v` and `-c` ?

So, `pretty-ms -v` would be equivalent to `pretty-ms --verbose` ?

Comment (sindresorhus) [inline readme.md]: Yea

Comment (icyflame) [inline readme.md]: I have sent a PR (#10)

**Pull request closed without merging**